src/conduits/RwaOutputConduit3.sol

```diff
@@ -54,8 +56,8 @@ contract RwaOutputConduit3 {
     /// @notice Dai output address
     address public to;
 
-    /// @dev This is declared here so the storage layout lines up with RwaOutputConduit.
-    mapping(address => uint256) private __unused_bud;
+    /// @dev Whitelist for addresses which can be picked. `bud[who]`
```

Comment:
Let's keep the NatSpec tag consistent (: 

```suggestion
    /// @notice Whitelist for addresses which can be picked. `bud[who]`
```

Reply:
Done